python/pytests/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode = auto
```

Comment:
Asyncio

Reply:
We don't have a pytest.ini -- we're following the newer pattern of putting the config in the `pyproject.toml`. See here https://github.com/kaskada-ai/kaskada/blob/main/python/pyproject.toml#L155